### ISSUE TITLE
jbuilder package dependency

### ISSUE THREAD

**Issue by wintersteiger** [FStarLang/FStar#1716]
My opam says:

```
=-=- jbuilder.transition installed successfully -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
=> Jbuilder has been renamed and the jbuilder package is now a transition package. Use the dune package instead.
```

**Comment by XVilka** [comment]
Note that the priority is not that low, since jbuilder transition period ends in a few days: https://dune.build/blog/second-step-deprecation/

The support for the jbuilder binary and jbuild files will be discontinued in July 2019. If you haven’t switch your project to dune already, now is a good time to do it.

To do that, you can either follow the [migration’s guide](https://dune.readthedocs.io/en/latest/migration.html), or use the newly added dune upgrade command that does it for you.

**Comment by XVilka** [comment]
I also noticed there is a Makefile used here https://github.com/FStarLang/FStar/blob/master/src/ocaml-output/Makefile
This part also can be ported to Dune build system, in fact, it will make F* building easier on Windows systems.

**Comment by tahina-pro** [comment]
I wrote an experiment report on building F* with dune on the F* GitHub wiki: https://github.com/FStarLang/FStar/wiki/Building-F*-with-dune%3F-(and-esy%3F)

**Comment by tahina-pro** [comment]
Fixed by #2815
